docs/src/app/components/pages/components/FlatButton/ExampleIcon.js

```diff
@@ -23,6 +23,7 @@ const FlatButtonExampleIcon = () => (
     <FlatButton
       href="https://github.com/callemall/material-ui"
       secondary={true}
+      target="_blank"
```

Comment:
I wouldn't worry about the security issue here. On the `next` branch we have an eslint rule for this.